mix.exs

```diff
@@ -17,10 +23,9 @@ defmodule Chronic.Mixfile do
 
   defp deps do
     [
-      {:earmark, "~> 1.2.6", only: :dev},
-      {:ex_doc, "~> 0.19", only: :dev},
+      {:ex_doc, ">= 0.0.0", only: :dev, runtime: false},
```

Comment:
I think it’d be better here to at least specify a minimum version.

Reply:
> I think it’d be better here to at least specify a minimum version.

Done. Set to `0.25.0`.